spec/index.html

```diff
@@ -378,7 +378,7 @@ <h2>Basic Concepts</h2>
     <p>
       Can be serialized as YAML as follows.
       Note that entries
-      starting with  `@` need to be enclosed in quotes or escaped because
+      starting with  `@` need to be enclosed in quotes because
       `@` is a reserved character in YAML.
```

Comment:
My read of [the YAML spec](https://yaml.org/spec/1.2.2/) with some reference to external docs like [Getting started with yaml, § Escaping Characters](https://riptutorial.com/yaml#escaping-characters) indicates that escaping *is* an option, though it's not being used here.

I believe it appropriate to mention that option in this side-note on this example, even though escaping is not being used here, because it can otherwise appear that we're saying that escaping is not an option.

A possible rephrasing of the note --

```suggestion
      Note that entries starting with `@` need to be
      enclosed in quotes (as shown in this example),
      or the `@` needs to be escaped,
      because `@` is a reserved character in YAML.
```

Reply:
@TallTed see YAML §5.7

> escape sequences are only interpreted in [double-quoted scalars](https://yaml.org/spec/1.2.2/#double-quoted-style).

In particular my preliminary tests with pyyaml e.g.

```
\u0040context: http://foo.bar
```

result in an error because it's equivalent to 

```
@context: http://foo.bar
```

I think we can write a testlist and ask to YAML folks for details.

Reply:
Ah, I think I see.  Looks like I didn't read deeply enough.  YAML has a longer learning curve than I anticipated. :-/

I think there are some other mentions of escaping in the docs which may also need to be eliminated or edited carefully by someone more deeply versed in YAML subtleties than I.

Reply:
@ioggstream —
> In particular my preliminary tests with pyyaml e.g.
>
>  ```\u0040context: http://foo.bar```

I wonder if that didn't fail because `\u0040` is an _encoding_ of `@`, rather than an _escape_, which I think would be `\@` (or *possibly* `\\\u0040` [i.e., double-backslash to escape the leading backslash, then another backslash to start the unicode encoding -- in other words, equivalent to `\@`]).